metapool/igm.py

```diff
@@ -0,0 +1,155 @@
+import os
+
+from datetime import datetime
+
+from openpyxl import load_workbook
+from openpyxl.styles import PatternFill
+
+_IGM_ROW_OFFSET = 25
+
+
+def _property_maker(obj, name, cell, default):
+    def fget(self):
+        return getattr(self, '_' + name)
+
+    def fset(self, value):
+        if value is not None:
+            self._sheet[cell] = value
+        setattr(self, '_' + name, value)
+
+    # set the default values
+    setattr(obj, '_' + name, default)
+    if default is not None:
+        getattr(obj, '_sheet')[cell] = default
+
+    setattr(type(obj), name, property(fget, fset))
+
+
+class IGMManifest(object):
+    """Interface to write an IGM manifest file
+
+    Attributes
+    ----------
+    submission_date: default today's date
+        When the sample sheet is submitted.
+    institute: default Knight Lab
+        Institution or laboratory.
+    pi_name: default Dr. Knight
+        Name of the PI.
+    pi_email: default mackkenzie.m.bryant@gmail.com
```

Comment:
mackkenzie.m.bryant@gmail.com isn't Rob's email. Is this okay, and should the field be renamed? Also, there's only one 'k' in Mackenzie's email address.

Reply:
In theory, yes but in practice MacKenzie's e-mail is the one that we want here.

Reply:
ok. we should still fix the email address to be 'mac**k**enzie.m.bryant' instead of 'mac**kk**enzie.m.bryant' though, right?

Reply:
Fixed! Looks like I messed up the formatting of the docstring so I changed this.